user_guide/intro_nav_sidebar.rst

```diff
@@ -0,0 +1,254 @@
+
+|
+
+
+
+You can:
+
+- Refresh the content (e.g. load the most actual emails or tasks) with |BRefresh|
+
+- Get to the widget settings with |IcSettings|
+
+- Delete the widget from the sidebar with |IcDelete|
+
+- Close the widget with |BCrLOwnerClear|  (appears only when the panel is minimized)
+
+
+.. hint::
+
+    You can add several widgets of the same type and define different settings for them. For example, you can keep notes 
+    with different content, or email widgets for different folders.
+
+    
+Default OroCRM Widgets
+----------------------
+
+This section describes the three widgets available in OroCRM out of the box.
+
+The "Recent Emails" Widget 
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The widget serves to inform you about new and unread emails and provides a convenient way to access them.
+
+|
+ 
+.. image:: ./img/navigation/sidebar/sb_emails.png
+
+|
+
+
+From the widget you can:
+
+- Access the *"All Emails"* page and see all your emails available in OroCRM: Сlick the *"My emails"* link (1).
+
+- Mark all the visible emails as read: Click the :guilabel:`Mark Visible As Read button` at the top (2).
+
+- Access any of the displayed emails: Click the Email header.(e.g. 3)
+
+- Mark any of the displayed emails as read: Click the envelope icon by the letter (e.g. 4)
+
+- Reply all, reply or forward the emails: All the actions are available as links below the email header, when the widget 
+  is expended from a minimized tray (e.g. 5). (If the panel bar is expended, only the default action is available).
+  
+- Get to the view page of the user from whose mailbox the emails has been sent (if available): click the user name below 
+  the letter (e.g. 6)
+
+  
+Configure the "Recent Emails" Widget 
+""""""""""""""""""""""""""""""""""""
+
+Click the |IcSettings| icon to configure the widget. The settings window will emerge:
+
+
+|
+
+.. image:: ./img/navigation/sidebar/sb_emails_set.png
+
+|
+
+- The widget can be configured to display all the emails from your OroCRM mailbox or from any specific folder or mailbox 
+  synchronized (1). 
+  
+- You can change the number of emails displayed (2).
+
+- You can define the default action shown if the sidebar panel has been expended (3).
+
+.. hint::
+
+    To make sure you don’t miss some particularly important emails, configure your mailbox to sort them in a specific 
+    folder, configure the widget to display this folder and place it at the top of your sidebar panel.
+  
+  
+Sticky Note
+^^^^^^^^^^^
+
+Sticky notes are a great way to keep reminders and memos, and you can keep them straight on your OroCRM sidebar panel.
+
+Click the |IcSettings| to change the content of a note.
+
+|
+
+.. image:: ./img/navigation/sidebar/sb_note.png
+
+|
+
+You can keep any number of sticky notes on the panel. If the panel is expanded, you can minimize some of the notes, and 
+keep others (for example, those that are due today) visible. Once the note is not acute anymore, you can delete it 
+easily.
```

Comment:
Edit: You can keep any number of sticky notes in the panel. If the panel is expanded, you can minimize some of the notes, and keep others visible (such as those that are due today). When you don't need the note anymore, you can easily delete it.